rootfs/Dockerfile

```diff
@@ -0,0 +1,20 @@
+FROM quay.io/deis/base:0.3.0
+
+ENV VERSION=0.3.8.linux-amd64.go1.6.2
+# Add nsq user and group
+RUN adduser --system \
+	--shell /bin/bash \
+	--disabled-password \
+	--home /opt/nsq \
+	--group \
+	nsq
```

Comment:
This `RUN` directive can be combined with the one below for one less layer in the image.